validator-auction-rest-server/lib/EthEventsClient.js

```diff
@@ -30,12 +30,19 @@ export default class EthEventsClient {
             this.getAuctionDeploymentParameters()
         ])
 
+        if(!deploymentParams || !auctionStart) {
```

Comment:
If the auction has not started yet, we can still already show some infos about the auction, so I think we should include all information, which are available even before the start of the auction (number of slots, length, start price )